configs/_base_/datasets/kitti-mono3d.py

```diff
@@ -0,0 +1,101 @@
+dataset_type = 'KittiMonoDataset'
```

Comment:
Remove this config because there are still some pipelines not implemented yet.

Reply:
remove the file to another PR